nemo_skills/evaluation/metrics/arena_metrics.py

```diff
@@ -120,12 +116,12 @@ def get_metrics(self):
 
         metrics = {'num_entries': self.total}
         metrics.update(get_aggregate_score(self.scores))
-        metrics['avg_response_length'] = self.lengths / self.total
+        if self.lengths > 0:
+            metrics['avg_response_tokens'] = int(self.lengths / self.total)
         return {self.agg_mode: metrics}
 
     def reset(self):
+        super().reset()
         self.scores = []  # list of lists
         self.lengths = 0
-        self.total = 0
-        # Set automatically
-        self.agg_mode = "greedy"
+        self.agg_mode = "pass@1"
```

Comment:
* This function might also throw the `NotImplementedError` since the `_get_score_dict` function is not there.
* Also, it does not use the new base metrics class to compute the pass@1